src/router/index.js

```diff
@@ -25,6 +25,11 @@ const router = createRouter({
       // which is lazy-loaded when the route is visited.
       component: () => import('../views/GameLibraryView.vue')
     },
+    {
+      path: '/game/:id',
```

Comment:
one final request, can you change the path to `/library/:id` since the user gets to the detail page from the library page

(or maybe we could change the library route to `/games` and then have this one be `/games/:id`)